root/etc/cont-init.d/30-config

```diff
@@ -254,6 +254,10 @@ if [ -n "$PEERS" ]; then
   else
     echo "**** External server address is set to $SERVERURL ****"
   fi
+
+  # Exit if SERVERURL not set
+  [ -z "$SERVERURL" ] && { echo "**** ERROR: External server address not found at my.dappnode/global-envs/HOSTNAME ****"; exit 1}
```

Comment:
This is just init script, not sure that `exit 1` will kill whole container.

Reply:
Adding `set -e` and having the propert restart policy should be enough right?

Reply:
Because everything is run by https://github.com/just-containers/s6-overlay#customizing-s6-behaviour it seems that we need to set environment variable `S6_BEHAVIOUR_IF_STAGE2_FAILS `.

Reply:
nice catch! so what should we do to includ this?
- env `S6_BEHAVIOUR_IF_STAGE2_FAILS`  only in this script or in `docker-compose.yml`
- `set -e`
- exit 1
?

Reply:
I would set it in Dockerfile so UI isn't spammed with all kinds of env variables.

Reply:
exit on error would require more research for next release https://github.com/dappnode/DNP_WIREGUARD/issues/20